generators/common.go

```diff
@@ -88,16 +100,47 @@ func templateFromManifest(cfg *GenerateCfg) (contractTmpl, error) {
 			var typeStr = cfg.ParamTypeConverter(method.Parameters[i].Type)
 
 			mtd.Arguments = append(mtd.Arguments, paramTmpl{
-				Name: name,
-				Type: typeStr,
+				Name:    name,
+				Type:    typeStr,
+				TypeABI: (smartcontract.ParamType).String(method.Parameters[i].Type),
 			})
 		}
 		mtd.ReturnType = cfg.ParamTypeConverter(method.ReturnType)
+		mtd.ReturnTypeABI = (smartcontract.ParamType).String(method.ReturnType)
 		ctr.Methods = append(ctr.Methods, mtd)
 	}
+
+	for _, event := range cfg.Manifest.ABI.Events {
+		name := event.Name
+
+		evt := eventTmpl{
+			Name:    name,
+			NameABI: event.Name,
```

Comment:
This fields seem to contain the same value as `Name` above it and is also not used in the `EVENTLISTENER` template, so I'm wondering why this exists?

Reply:
My bad, I forgot to delete this field